### PR TITLE
Sass auto migrator

### PR DESCRIPTION
**Pull request by n3ssi3**

### CODE REVIEW

**Comment by n3ssi3** [comment]
sorry didn't mean to do that

**Comment by sciborrudnicki** [comment]
Good. I will use it here https://github.com/angular-package/spectre.css/tree/1.0.x

**Review by sciborrudnicki — COMMENTED** [review]
Checked.

File: src/_layout.scss [hunk 1]
```diff
@@ -1,12 +1,14 @@
...
+
 .container {
   margin-left: auto;
   margin-right: auto;
   padding-left: $layout-spacing;
   padding-right: $layout-spacing;
   width: 100%;
 
-  $grid-spacing: ($layout-spacing / ($layout-spacing * 0 + 1)) * $html-font-size;
+  $grid-spacing: math.div($layout-spacing, $layout-spacing * 0 + 1) * $html-font-size;
```
[the remark this answers is not in the document]
Comment (sciborrudnicki) [reply src/_layout.scss:11]: This is interesting,

`($layout-spacing / ($layout-spacing * 0 + 1)).`

Originally $layout-spacing is equal to `0.4`, so `(0.4 / (0.4 * 0 + 1)) = 0.4`
Let the `$layout-spacing` be equal to `10.5`, then `(10.5 / (10.5 * 0 + 1)) = 10.5`

What is the point of this.

[the remark this answers is not in the document]
Comment (sciborrudnicki) [reply src/_layout.scss:11]: Understood.

[the remark this answers is not in the document]
Comment (sciborrudnicki) [reply src/_layout.scss:11]: @use 'sass:math';

@function strip-units($number) {
  @return math.div($number, ($number * 0 + 1));
}

**Pull request closed without merging**